### DISCUSSION TITLE
awake android

### DISCUSSION THREAD

**Discussion by hank09** [rustdesk/rustdesk#963] [Q&A]
安卓锁屏的情况下不能直接唤醒么？做了一些测试，必须手动解锁之后才能正常操作

**Comment by Heap-Hop** [comment]
#550
#729

Please provide your Android device info and rustdesk apk version.

> 做了一些测试，必须手动解锁之后才能正常操作

BTW, did you open the `Input Control` permission during your test?

**Comment by Jesn** [comment]
在Mac上链接华为（鸿蒙系统），唤醒黑屏，无法输入解锁密码
